src/main/java/uk/gov/hmcts/reform/pip/data/management/service/ValidationService.java

```diff
@@ -205,4 +205,10 @@ private static boolean isNullOrEmpty(Object header) {
         return header == null || header.toString().isEmpty();
     }
 
+    public void containsForbiddenCharacter(String inputField) {
+        if (inputField.indexOf('<') != -1) {
```

Comment:
I think it is better to check for a opening angular bracket followed by some text then a closing bracket, just like how Chris was doing in the list type validation schema to specifically test for html tags. Regular expression will be suitable here